examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/observe_with_check.py

```diff
@@ -30,38 +26,31 @@
     name=AIRFLOW_INSTANCE_NAME,
 )
 
+dbt_freshness_checks = build_last_update_freshness_checks(
+    assets=[DBT_DAG_ASSET_KEY],
+    lower_bound_delta=timedelta(hours=1),
+    deadline_cron="0 9 * * *",
+)
 
-def freshness_defs() -> Definitions:
-    dbt_freshness_checks = build_last_update_freshness_checks(
-        assets=[DBT_DAG_ASSET_KEY],
-        lower_bound_delta=timedelta(hours=1),
-        deadline_cron="0 9 * * *",
-    )
-    return Definitions(
-        asset_checks=dbt_freshness_checks,
-        sensors=[
-            build_sensor_for_freshness_checks(
-                freshness_checks=dbt_freshness_checks,
-            )
-        ],
-    )
-
-
-defs = build_defs_from_airflow_instance(
-    airflow_instance=airflow_instance,
-    defs=Definitions.merge(
-        dag_defs(
-            "rebuild_iris_models",
-            task_defs("load_iris", Definitions(assets=specs_from_lakehouse(csv_path=CSV_PATH))),
-            task_defs(
-                "build_dbt_models",
-                Definitions(assets=build_dbt_asset_specs(manifest=dbt_manifest_path())),
-            ),
-        ),
-        lakehouse_existence_check_defs(
+defs = Definitions(
+    assets=assets_with_task_mappings(
+        dag_id="rebuild_iris_models",
+        task_mappings={
+            "load_iris": specs_from_lakehouse(csv_path=CSV_PATH),
+            "build_dbt_models": [jaffle_shop_assets],
+        },
+    ),
```

Comment:
I kinda wonder if we should split this out into it's own line and call it "mapped_assets = ..." or something. dag_defs and task_defs had cleaner syntax and was thus easier to parse. This requires about 2-3 looks I'd say to understand what's happening.

Reply:
Hmmm I'm not following exactly. Can you provide an exact example of what you are suggesting?

Reply:
Move the call that maps the airflow metadata to its own line of code rather than keeping it inline.